matRad_calcLateralParticleCutOff.m

```diff
@@ -121,7 +126,7 @@
                     r_cut = interp1(cumAreaUnique,r_mid(IdxUnique),currCutOffLevel);
                 end
             catch
-                error('code goes here');
+                error('error in matRadcalcLateralParticleCutOff - cannot determine cut off');
```

Comment:
I don't like the nesting of this if else try catch block.

Shouldnt it be

if currC....
  do something
else
  try
   do something
  catch
   error...
  end
end